amplitude.py

```diff
@@ -20,6 +22,21 @@
 # specifying wbits=0 but that didn't work for me locally. This did.
 ZLIB_HEADER_SIZE = 32
 
+def kms_decrypt_env(key):
+    """Decrypt environment variable"""
+    return kms_decrypt(b64decode(os.environ[key]))
+
+def kms_decrypt(encrypted_data):
+    """Decrypt KMS variables"""
+    res = boto3.client("kms").decrypt(
+        CiphertextBlob=encrypted_data,
+    )
+    return res["Plaintext"].decode("utf-8")
+
+if os.environ["LAMBDA_TASK_ROOT"]:
```

Comment:
This raises a `KeyError` if `LAMBDA_TASK_ROOT` is not defined. We should avoid that so we can still test locally:

```py
if "LAMBDA_TASK_ROOT" in os.environ:
```